examples/agentInteractionsDemo/app.js

```diff
@@ -207,9 +209,17 @@ new Vue({
                                 .then((conv) => {
                                     let newConv = conv;
                                     newConv.customer = conv.participants.find((part) => {
```

Comment:
In the scenario where this `find` doesn't yield a customer (not sure if possible), should we be a bit more robust and fallback to `{ name: 'Unknown' }` so that the conversation can still render?

Perhaps something like:
```
newConv.customer = conv.participants.find(...) || { name: 'Unknown" };
```

Reply:
Good idea! It would definitely make it the demo more robust.